.github/FUNDING.yml

```diff
@@ -0,0 +1,12 @@
+## These are supported funding model platforms
```

Comment:
Don't merge this file until its configured correctly - we don't need a placeholder.